python-template/{{cookiecutter.placeholder_repo_name}}/tox.ini

```diff
@@ -0,0 +1,72 @@
+[tox]
+envlist = py{35,36,3738}
+
+[doc8]
+max-line-length = 120
+
+[pycodestyle]
+exclude = .git,.tox
+max-line-length = 120
```

Comment:
Our [Python style guide](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#syntax-and-organization) recommends 79 for docstrings.

Reply:
doesn't this setting control code length rather than docstrings

Reply:
nevermind, found --max-doc-length=n